owa/app/js/home/home.html

```diff
@@ -1,3 +1,6 @@
+<div>
+    <a class="button" href="#/mapping"><i class="icon-plus"></i>&#160;Create Mapping</a>
```

Comment:
@dkayiwa I thought you are waiting for @rkorytkowski opinion, but I can merge it :)